src/extensions/default/bramble/lib/Tutorial.js

```diff
@@ -9,24 +9,43 @@ define(function (require, exports, module) {
     var Path            = Filer.Path;
 
     var PostMessageTransport = require("lib/PostMessageTransport");
-
+    var UI                   = require("lib/UI");
+    
     // Whether or not we're overriding the preview with a tutorial
     var _tutorialOverride;
 
     // When we want to override, in order to get the tutorial to
     // show, we need to force a reload for the preview.
     var _forceReload;
 
+    // Preserves the mobile view state
+    var _shouldRestoreMobileView = false;
+
     function setOverride(val) {
         _tutorialOverride = !!val;
+        
+        // If mobile view is on we need to preserve it.
+        if (UI.getPreviewMode() === "mobile") {
+            _shouldRestoreMobileView = true;
+        }
 
         // We need to reload the browser preview. If we're going to override,
         // we can simply reload the current live dev instance.  If not, we
         // need to properly reset live dev so it points at the project's main file.
+        
         if(_tutorialOverride) {
             _forceReload = true;
+            UI.showDesktopView(true);  // Always show in desktop mode
```

Comment:
@flukeout i think you meant `true` instead of `false` and i removed the reload line.... the brief flash is not seen now.....please review the following changes.